app/views/layouts/application.html.erb

```diff
@@ -104,19 +104,17 @@
       </div>
     <% end %>
     <%= yield %>
-    <% if FeatureFlag.enabled?(:runtime_banner) || ENV['E2E'] == 'true' %>
-      <div class="Browser-iOS">
-        <div class="runtime-banner">
-          <a href="<%= URL.deep_link(request.path) %>" target="_blank" class="flex items-center flex-1">
-            <%= inline_svg_tag("logo-forem-app.svg", aria_hidden: true, class: "crayons-icon crayons-icon--default", width: 32, height: 32) %>
-            <span class="ml-2">Open with the Forem app...</span>
-          </a>
-          <button type="button" class="runtime-banner__dismiss crayons-btn crayons-btn--ghost crayons-btn--icon crayons-btn--inverted crayons-btn--s pl-2">
-            <%= inline_svg_tag("x.svg", aria: true, class: "crayons-icon", title: "Dismiss banner: Open with the Forem app") %>
-          </button>
-        </div>
+    <div class="Browser-iOS">
```

Comment:
Interesting...is `Browser-iOS` a class that comes from somewhere else? It just caught my eye because it's formatted so oddly (not like usual CSS) so I was wondering if it's a class that comes from/is defined elsewhere

Reply:
`app/assets/javascripts/initializers/runtime.js` see https://github.com/forem/forem/pull/13007 I think the distinct formatting makes sense, as this is used for our runtime content filter not "classic" CSS.

Reply:
That's correct, the definition comes from [RFC 27 (Implementation)](https://github.com/forem/rfcs/blob/315ada9bbd6a033f831a33286b0066d7e8553c74/text/0027-runtime-content-filter.md#designimplementation).

We could also consider adding an extra word to them (i.e. `Browser-iOS-filter` or `Browser-iOS-only`). That would make them a bit more self-explanatory (for those without context), despite being more verbose it _could_ be valuable.

I don't have strong opinions on this though (I'm okay with leaving it as-is) but I'm open to revisiting it anytime 🙂

Reply:
Same as you, I don't feel strongly about it but I do like how declarative and self-explanatory  `Browser-iOS-only` is.

Reply:
+ 1 on `Browser-iOS-only` for people that are encountering it for the first time

Reply:
Thanks for the explanation, folks! (Apparently even though I _reviewed_ that RFC I promptly forgot about why we named that class that way 😛)

I always will be the one to vote for explicit naming when possible, so `Browser-iOS-only` definitely seems more clear and self-explanatory 👍🏽

Reply:
To confirm here: I updated these classes to include the `-only` suffix 😃